spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/SpatialConvolution.scala

```diff
@@ -90,6 +90,8 @@ class SpatialConvolution[T: ClassTag](
   @transient
   protected var results: Array[Future[Unit]] = null
 
+  setTrainable(propagateBack)
```

Comment:
in SpatialConvolution, if propagateBack = false, the gradInput is not computed.

if trainable = false, accGradParameters is not called.